server/index.js

```diff
@@ -6,8 +6,8 @@ const compression = require('compression');
 const dot = require('dot');
 
 const api = require('./api/index');
-
-
+const csp = require('helmet-csp');
+const securityHelper = require('./utils/secureUtil');
```

Comment:
What's supposed to be on this file? I tried to start the project, but it was complaining it doesn't exist.

Reply:
Oops wait, looks like my branch has missed few commits. Will resolve and push them.

Reply:
@nglgzz Could you please try now? I have added the utils files.  I'm rebasing my codebase, because even your commits are showing up in my PR.

Reply:
Okay it works now. I had to add the following to the directives, since I forgot to mention we're sending requests to Auth0 for authentication.
```
connectSrc: [
          "'self'",
          'https://learn-anything.auth0.com',
        ]
```

Everything seems to work fine, but I keep getting this error on console:
```
EventSource's response has a MIME type ("text/html") that is not "text/event-stream". 
Aborting the connection.
```
Any idea why? The error it thrown every few seconds.

Reply:
@nglgzz Yeah, even I noticed it now. Will debug and let u know, may be we need to remove each headers from CSP and see which one is making it go and figure out why its happening.

Reply:
@nglgzz Just debugged it, looks like its some sort of issue with webpack hot reload. If I add something to the config like : 

```
'webpack-hot-middleware/client?path=http://localhost:3000/__webpack_hmr'
```

i.e removing the `reload`, so my guess here is that, `reload` is somehow sending some wrong data. Give me some time, as we need to clearly understand , who is causing this issue, before we do a fix. Any thoughts?

Reply:
Yeah makes sense that the problem is there, as the hot reloading is the only thing sending data every few seconds. Could it be that something is changing its data type? I found this related issue https://github.com/glenjamin/webpack-hot-middleware/issues/26 is it of any help?

Reply:
@nglgzz Yeah you are right. So, when I run it in production, there is no hot reload and the error is gone. May be, we can move all these headers only for production mode, that would help as well.

Reply:
Oh yeah, that would definitely work 👍

Reply:
@nglgzz Made the check to be happening only on production, can you just check from your side and let me know? Once confirmed, I can squash my commits and give the changes in a single shot. Have a good day!

Reply:
Hey @antoaravinth, just tried and it was still throwing the error, but I managed to get rid of it by adding back the `app.use(webpackHotMiddleware(compiler));` line. There was also another GA url that was missing, so I added that too. 

One small detail and then everything is good for merging. Can you change the indentation to be 2 spaces instead of 4? As that's what we're using for the whole project.

Reply:
@nglgzz Sure, will do. But strange how come I don't get the error without your changes! Anyways thanks for your fix. Will change the indentation and push by tomorrow.